jenkins/ci.suse.de/cloud-maintenance.yaml

```diff
@@ -60,18 +60,16 @@
           cloudsource: GM7+up
           ses_enabled: false
           update_after_deploy: false
-          # magnum not fully passing yet
           tempest_filter_list: "\
             keystone,swift,glance,cinder,neutron,nova,ceilometer,fwaas,\
-            trove,aodh,heat,manila,lbaas"
+            trove,aodh,heat,magnum,manila,lbaas"
```

Comment:
I would recommend not enabling those tests here until we have released those changes or they will keep breaking the maintenance-gating job.

Reply:
@flaviodsr you think I can remove maintenance part and get this merged ? and later add part for maintenance jobs